### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by yzhang90**

### CODE REVIEW

**Review by ankushdesai — CHANGES_REQUESTED** [review]
Can you please rebase this pull request with the new master and make sure that the regressions passes?

File: .github/workflows/dotnetcore.yml [hunk 1]
```diff
@@ -11,6 +11,15 @@ jobs:
...
       with:
         dotnet-version: 3.1.100
+    - name: Set up JDK
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Build P runtime for Java
+      run: |
+        pushd Src/PRuntimes/RvmRuntime
+        mvn install
```
[the remark this answers is not in the document]
Comment (yzhang90) [reply .github/workflows/dotnetcore.yml:21]: yes. we add a new test runner and when you run `dotnet test --configuration Release`, it should run all the tests including the rvm test. If you want to rvm test specifically, you can run with `dotnet test --filter FullyQualifiedName~UnitTests.RvmRuntimeRegression`

**Review by ankushdesai — CHANGES_REQUESTED** [review]
Everything looks great! 

One minor comment:
Can you create a new step in each github action that runs the rununittests.py as a commend directly instead of through the .net code. 

You can call it Test-RVM step.

**Comment by virgil-serbanuta** [comment]
> One minor comment:
> Can you create a new step in each github action that runs the rununittests.py as a commend directly instead of through the .net code.
> 
> You can call it Test-RVM step.

I think that's done, does it look reasonable now?

**Review by ankushdesai — APPROVED** [review]
Thanks @yzhang90 and @virgil-serbanuta, this looks great now!

**Pull request merged**